hw/ip/clkmgr/dv/cov/clkmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,154 @@
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_hmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_kmac_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "2972535896 3274445021"
+INSTANCE: tb.dut.u_reg.u_clk_hints_status_clk_main_otbn_val
+ANNOTATION: "VC_COV_UNR"
+Condition 1 "2397158838" "(wr_en ? wr_data : qs) 1 -1" (1 "0")
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_main_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_io_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_io_div2_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_io_div4_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "3864567832 3302102175"
+INSTANCE: tb.dut.u_usb_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Block 1 "254557145" "if ((!rst_ni))"
+ANNOTATION: "VC_COV_UNR"
+Block 3 "1282307173" "gen_no_flops.unused_logic <= mubi_i;"
+CHECKSUM: "215202837 3193272610"
+INSTANCE: tb.dut.u_io_meas.u_meas.gen_clk_timeout_chk.u_timeout_ref_to_clk
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "3003057152" "(!rst_ni)" (4) "(!rst_ni) 0,0,0,0"
+CHECKSUM: "215202837 3193272610"
+INSTANCE: tb.dut.u_io_div2_meas.u_meas.gen_clk_timeout_chk.u_timeout_ref_to_clk
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "3003057152" "(!rst_ni)" (4) "(!rst_ni) 0,0,0,0"
+CHECKSUM: "215202837 3193272610"
+INSTANCE: tb.dut.u_io_div4_meas.u_meas.gen_clk_timeout_chk.u_timeout_ref_to_clk
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "3003057152" "(!rst_ni)" (4) "(!rst_ni) 0,0,0,0"
+CHECKSUM: "215202837 3193272610"
+INSTANCE: tb.dut.u_main_meas.u_meas.gen_clk_timeout_chk.u_timeout_ref_to_clk
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "3003057152" "(!rst_ni)" (4) "(!rst_ni) 0,0,0,0"
+CHECKSUM: "215202837 3193272610"
+INSTANCE: tb.dut.u_usb_meas.u_meas.gen_clk_timeout_chk.u_timeout_ref_to_clk
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "3003057152" "(!rst_ni)" (4) "(!rst_ni) 0,0,0,0"
+CHECKSUM: "3864567832 1223333544"
+INSTANCE: tb.dut.u_main_root_ctrl.u_scanmode_sync
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "455461571" "(!rst_ni)" (0) "(!rst_ni) 1"
```

Comment:
It seems these are because the prim_mubi4_syncs with no flops tie rst_ni to a constant. This is non-synthesized code so will be removed in a subsequent PR.